vagga.yaml

```diff
@@ -0,0 +1,36 @@
+minimum-vagga: v0.7.0
+containers:
+  pip-freeze:
+    setup:
+    - !Alpine v3.4
+    - !Py3Install
+      - pip
+      - 'Django >=1.8'
```

Comment:
Probably yes, it may also lock minor version, but allow updating patch version. Like `>=1.8,<1.9`